structure_tests/structure_test_v1.go

```diff
@@ -186,7 +186,21 @@ func ProcessCommand(t *testing.T, envVars []EnvVar, fullCommand []string, checkO
 		} else {
 			t.Fatalf("Error running setup/teardown command: %s.", err)
 		}
-		exitCode = err.(*exec.ExitError).Sys().(syscall.WaitStatus).ExitStatus()
+
+		exitErr, ok := err.(*exec.ExitError)
+		if !ok {
+			exitErr, ok := err.(*exec.Error)
+			if ok {
+				t.Logf("Command %s failed to run! Error: %s", exitErr.Name, exitErr.Error())
+				t.Logf("Attempting to run in shell mode.")
```

Comment:
I don't think there's a way to tell if we need to run it in shell mode (without trying to run it first), but we could ask the users to specify. My thinking with doing it this way is that it's a little less confusing for the test writers, and a bit more abstracted to them. WDYT?